apps/web/src/views/PositionManagers/components/StakedAssets.tsx

```diff
@@ -57,6 +71,10 @@ export const StakedAssets = memo(function StakedAssets({
           <Text color="text" fontSize="1.5em" bold>
             ~${totalAssetsInUsd.toFixed(2)}
           </Text>
+          <Text color="textSubtle" fontSize="0.75em">
+            (~{totalAssetInSingleDepositTokenAmount.toFixed(6)}{' '}
+            {isSingleDepositToken0 ? currencyA.symbol : currencyB.symbol})
```

Comment:
Can we make it like this, can use useMemo show the final text.

const symbol = isSingleDepositToken0 ? currencyA.symbol : currencyB.symbol}
{~`${totalAssetInSingleDepositTokenAmount.toFixed(6)}} ${symbol}`}

Reply:
Fixed.